<?xml version="1.0"?>
<launch>


 <include file="$(find openni2_launch)/launch/openni2.launch">
    <arg name="camera" value="bottom_camera"/>
    <arg name="device_id" value="#1"/> 
    <arg name="depth_camera_info_url" value ="file://$(find help_indicators)/calibration/depth_PS1080_PrimeSense.yaml" />
 </include>
	

  <node name="help_indicators" pkg="help_indicators" type = "help_indicators" respawn = "true" output="screen">
  <param name="kinect_max_dist" type="double" value="5" />
  <param name="kinect_min_dist" type="double" value="0" />
  <param name="store_depth" type="bool" value="false" />		
 </node>	

 <node name="help_transformers" pkg="help_indicators" type="help_transformers" respawn="true" output="screen"/>
 <node name="controller" pkg="tangy_move" type="controller" respawn="true"/>
 
</launch>
